nex/matchmake-extension/get_friend_notification_data.go

```diff
@@ -1,36 +1,29 @@
 package nex_matchmake_extension
 
 import (
-	"fmt"
-
 	nex "github.com/PretendoNetwork/nex-go"
 	"github.com/PretendoNetwork/wiiu-chat-secure/database"
 	"github.com/PretendoNetwork/wiiu-chat-secure/globals"
 
 	matchmake_extension "github.com/PretendoNetwork/nex-protocols-go/matchmake-extension"
 	"github.com/PretendoNetwork/nex-protocols-go/notifications"
+	notifications_types "github.com/PretendoNetwork/nex-protocols-go/notifications/types"
 )
 
 func GetFriendNotificationData(err error, client *nex.Client, callID uint32, uiType int32) {
-	fmt.Println(client.PID()) // DEBUG
-
-	/*if !isUserAllowed(client.PID()) {
-		globals.NEXServer.Kick(client)
-		// get outta here
-	}*/
-	// pls stay, whitelist is gone
-
-	dataList := make([]*notifications.NotificationEvent, 0)
+	dataList := make([]*notifications_types.NotificationEvent, 0)
 
 	caller, target, ringing := database.GetCallInfoByTarget(client.PID())
 
 	// TODO: Multiple calls. Wii U Chat can handle it, but we don't support it yet
 	if (caller != 0) && (target == client.PID()) && ringing {

```

Comment:
Unnecessary parentheses (I know this was not done by you but it showed in your PR so I'm commenting anyway)